### ISSUE TITLE
Accessibility issue | Table panel | Keyboard navigation | Screen reader

### ISSUE THREAD

**Issue by Darkavid** [grafana/grafana#84243]
**Steps to reproduce**:
1. Create a dashboard
2. Add a new panel
3. Set the visualization to 'Table'
4. Add some basic query to get some data
5. Save the panel
6. Make sure the focus is "before" the newly created Table panel
7. Use keyboard (e.g. via Tab key) to navigate and move focus

**Actual Result**:
Via the keyboard, I cannot move the focus to the table contents, specifically the individual rows.

**Expected Result**
Via the keyboard, I should be able to move the focus to the table contents, specifically the individual rows.

**Rationale**:
It is common that a Table panel contains many rows. Having no possibility to move the focus individually between the rows breaks the accessibility while using a screen-reader. Even if users are curious only about some particular information in a Table panel, they need to listen all table information presented in all rows and if they missed some, they need to listen all contents of the panel again for no gain at all.

**Relevant WCAG Criteria:**
[2.1.1 WCAG Criterion](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

**Environment**:
- Grafana version: Grafana v10.2.3 (1e84fede54)
- Data source type & version: Any (irrelevant)
- User OS & Browser: Windows 11 & Google Chrome (Version 122.0.6261.112)
